tasks/page/processIBOM.js

```diff
@@ -11,23 +10,25 @@ const omitIBOMBoards =
 
 if (require.main !== module) {
   module.exports = function(config, boardInfo) {
-    let kicadPcbFile
+    let pcbFile
     if (
       boardInfo.eda &&
-      boardInfo.eda.type === 'kicad' &&
+      (boardInfo.eda.type === 'kicad' || boardInfo.eda.type === 'eagle') &&
       boardInfo.eda.pcb != null
     ) {
-      kicadPcbFile = path.join(boardInfo.repoPath, boardInfo.eda.pcb)
+      pcbFile = path.join(boardInfo.repoPath, boardInfo.eda.pcb)
     } else if (boardInfo.eda == null) {
-      const kicadPcbPattern = path.join(boardInfo.boardPath, '**/*.kicad_pcb')
-      kicadPcbFile = globule.find(kicadPcbPattern)[0]
+      pcbFile = utils.findBoardFile(boardInfo.boardPath, 'kicad_pcb')
+    }
+    if (pcbFile === undefined) {
+      pcbFile = utils.findBoardFile(boardInfo.boardPath, 'brd', utils.checkEagleFile)
     }
     const omit = omitIBOMBoards.includes(boardInfo.boardPath.split('/').slice(1).join('/'))
-    if (kicadPcbFile == null || omit) {
+    if (pcbFile === undefined || omit) {
```

Comment:
Can it be `null` now? We try and pretend that `null` and `undefined` are the same thing as much as possible throughout the code base by using `== null` rather than `=== null` or `=== undefined`.

Reply:
I'll fix this. Javascript comparison semantics drive me to despair...